.github/workflows/create-milestone.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           quarter=$(date +"%q")
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31890&commitId=bdf92a04255ca03c2d0c684fea664a67d0deb76c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|6| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>